components/admin_console/webrtc_settings.jsx

```diff
@@ -137,6 +139,25 @@ export default class WebrtcSettings extends AdminSettings {
                     disabled={!this.state.enableWebrtc}
                 />
                 <TextSetting
+                    id='gatewayType'
```

Comment:
do we validate the value entered in this field? what are the possible values? what happens if I failed to type **Janus** and I type **jans** I think this should be a dropdown with the available options

Reply:
Yes a dropdown can work nicely, but is only useful in the admin UI. In the configuration file its a string value, so imho it can be a string value in the web as well. If an unknown value is used, the the Janus implementation is used.

Reply:
@longsleep Propose we use a dropdown for the UI as it's more consistent with our other System Console pages.

Although I do hear your point about it not applying to the configuration file, it'll be more forgiving as the admin cannot make a typo when using the UI.

Reply:
> @longsleep Propose we use a dropdown for the UI as it's more consistent with our other System Console pages.

Ok - will make the change during Hacktoberfest

Reply:
Pushed another commit which moves `GatewayType` to a dropdown. For some reason GitHub does not show it yet though ... Commit https://github.com/longsleep/mattermost-webapp/commit/93f45322dfbb5a1b56f3740aef7a51e83a92f76a should show up here.